tools/circleci/images.yml

```diff
@@ -5,3 +5,7 @@ trusty:
 xenial:
   base_image: buildpack-deps:xenial-scm
   extra_packages: virtualenv postgresql-9.5
+
+bionic:
+  base_image: buildpack-deps:bionic-scm
+  extra_packages: virtualenv postgresql-10
```

Comment:
This is mostly unrelated, but I suspect we no longer need the `virtualenv` in `extra_packages` anymore.

Reply:
See, for example, 0fabff6ddacf40de2de999eac43c8ddc11d8f4fe